### DISCUSSION TITLE
readme - cannot find (copyrights, credits, licenses, references)

### DISCUSSION THREAD

**Discussion by fzs1933** [Stellarium/stellarium#2023] [Q&A] [answered]
I would like to read about copyrights of make and use screenshots, videos, etc. I find it everywhere this: **_"The README file in the program folder contains all known credits, references and special licenses for images and data."_**. But my Stellarium folder does not contain readme, only in landscapes directory. I searched in the web, without result.
Stellarium 0.21.0
Where can I read it?
Thanks
Zsolt

**Comment by alex-w** [comment, marked as answer]
What you see here: https://github.com/Stellarium/stellarium ?

**Reply by gzotti** [reply]
I think that README used to be in the distribution as well. But at least on Windows hardly anybody looks in the program folder for copyright info or other files without known extension, The credits are now (also) in the User Guide, Appendix H.6. Indeed, the FAQ needs a little update.

**Reply by fzs1933** [reply]
Sorry, I was careless...
Thanks for your patient.
Zsolt

**Reply by alex-w** [reply]
I cannot give correct answer where README and other files are located without knowledge about target platform.

**Reply by fzs1933** [reply]
What do you mean by the target platform?
Do you mean the purpose I want to use Stellarium or the hardware and operating system?
Sorry for my incomprehension (I am not too practiced in english and this technical language).
But - I think you and gzotti gave the answer I wanted.

**Reply by gzotti** [reply]
Stellarium is multi-platform, and existence, position and file extension may differ whether you have a Windows, Mac or Linux system.

**Reply by fzs1933** [reply]
THanks, platform may mean purpose in hungarian... :)
I use Windows 10 64 bit if it matters

**Reply by alex-w** [reply]
Please check install directory of Stellarium ;)

**Reply by fzs1933** [reply]
Is it possible that I am blind? :)
![stellarium](https://user-images.githubusercontent.com/81897104/141110457-71e08e9a-ea6d-47bb-a208-b768cefa3a54.jpg)
So, I dont want to waste your time. As I wrote earlier, probably will be enough the users guide references and the github folder.
If not, I'll apply anyway... :)
Thanks your attentions and answers!

**Reply by alex-w** [reply]
Hmm... probably we have bug in scripts for Windows installer :( I'll check it, thanks for report!

**Reply by fzs1933** [reply]
Hi,
Just tried...
I downloaded the 0.21.2 and it is same. Stellarium folder does not contain readme file.

Regards
